CMakeLists.txt

```diff
@@ -209,6 +209,17 @@ set_property(TARGET ${PROJECT_NAME} PROPERTY POSITION_INDEPENDENT_CODE ON)
 # INSTALL
 #################################################################################
 
+# libmd.pc for pkg-config
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+   set(DEST_DIR /usr/local)
+else()
+    set(DEST_DIR ${CMAKE_INSTALL_PREFIX})
+endif()
+set(LIB -l${PROJECT_NAME}) # -lmd
+set(MD_VERSION ${MD_VERSION})
+configure_file(libmd.pc.in libmd.pc @ONLY)
+install (FILES libmd.pc DESTINATION /usr/lib64/pkgconfig)
```

Comment:
Use `lib/pkgconfig` instead of `/usr/lib64/pkgconfig`, otherwise `make install` step will not work properly when users use `-DCMAKE_INSTALL_PREFIX=/any/other/directory` to define where to install the library.

Reply:
@anselmolsm, it works regardless of CMAKE_ISNTALL_PREFIX settings, since this is an absolute path. And if we use lib/pkgconfig instead of the absolute path /usr/lib64/pkgconfig, users will be required to set PKG_CONFIG_PATH on Clear Linux even with the default setting builds, because it doesn't looks /usr/local/lib/pkgconfig to search pc files.

Reply:
It works regardless of CMAKE_INSTALL_PREFIX in terms of building, but make install won't install to the desired prefix.
Correct, PKG_CONFIG_PATH has to be set whenever the user installs to a non-default path. However Clearlinux packaging does set prefix to /usr (I mean, it does not use the default /usr/local prefix).